config/routes.rb

```diff
@@ -36,4 +36,15 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  post "/applications", to: "applications#create"
+  get "/applications/new", to: "applications#new"
+  get "/applications/:id", to: "applications#show", as: 'show_application'
+  post "/applications/:id", to: "pet_applications#create", as: 'create_pet_application'
+  patch "/applications/:id", to: "applications#update"
+
+  get "/admin/shelters", to: "admin/shelters#index"
+
+  get "/admin/applications/:id", to: "admin/applications#show"
+  patch "/admin/applications/:id", to: "admin/applications#update"
```

Comment:
This action seems to be updating the pet_application rather than the pet, so a better RESTful choice here would be
`patch "/applications/:app_id/pets/:pet_id", to: "pet_applications#update"`
Note that we don't need the `admin` prepended here because this is not an external route that a user goes to, but a route that is triggered from clicking a button on that page. Then, once updating the pet application, you can redirect to an admin path